http/headers/content-security-policy.json

```diff
@@ -588,7 +588,8 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": "33"
+                "version_added": "33",
+                "notes": "Before Firefox 58, this would be ignored in <code>Content-Security-Policy-Report-Only</code>"
```

Comment:
"Before Firefox 58, \<code\>frame-ancestors\<code\> is ignored in <code>Content-Security-Policy-Report-Only</code>."
Note the present tense, and the final full stop. (I've also replaced the this, but it si more a matter of taste)

Reply:
That reads a lot better. I wasn't sure if there was a standard for using "this" vs. the feature's name, but if it's acceptable to specifically call it out I'd rather do that.